packages/ERTP/src/issuer.js

```diff
@@ -14,11 +14,9 @@ import { makeInterface, ERTPKind } from './interfaces';
 import './types';
 
 /**
- * @param {string} allegedName
- * @param {AmountMathKind} [amountMathKind=MathKind.NAT]
- * @returns {IssuerKit}
+ * @type {MakeIssuerKit}
  */
-function makeIssuerKit(allegedName, amountMathKind = MathKind.NAT) {
+function makeIssuerKit(allegedName, amountMathKind = MathKind.NAT, decimals) {
```

Comment:
```suggestion
function makeIssuerKit(allegedName, amountMathKind = MathKind.NAT, decimals = undefined) {
```

Reply:
This was frowned upon by the typescript enforcement because number cannot be undefined. I actually had this at first and had to revert.